scripts/generateYaml.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
```

Comment:
what mechanism replaced this in subgraph yaml generation? When new local deployment is done, those yaml need to be re-generated to match the new addresses...